project/src/views/evaluations/evaluations.js

```diff
@@ -1,90 +1,81 @@
-const EvaluationsPage = function(options = {}) {
+
+const EvaluationsTableHeader = function (options = {}) {
+    const headings = options.headings.map(el => `<th class="evaluations-tbl-head evaluations-tbl-head-skin">${el}</th>`).join('');
+
+    return `
+      <tr>
+          ${headings}
+      </tr>
+      `;
```

Comment:
I like how returned contents are being indented but the closing might make more sense to be on the same column as the opening for easier readability.